client/connection/openvpn.go

```diff
@@ -35,7 +35,12 @@ func ConfigureVpnClientFactory(
 
 		statsSaver := bytescount.NewSessionStatsSaver(statsKeeper)
 		statsSender := bytescount.NewSessionStatsSender(mysteriumAPIClient, vpnSession.ID, signer)
-		statsHandler := bytescount.NewCompositeStatsHandler(statsSaver, statsSender)
+		asyncStatsSender := func(stats bytescount.SessionStats) error {
+			go statsSender(stats)
```

Comment:
In case of undelivered requests we will form a queue of hanging routines.
Can You create a ticket for smarted async middlewares.

Reply:
Created MYST-338